remotezip.py

```diff
@@ -252,4 +255,45 @@ def _get_position_to_size(self):
         return {a: b-a for a, b in pairwise(ilist)}
 
 
-
+def list_files(url, support_suffix_range, filenames):
```

Comment:
I would prefer to keep this function and the next private as they are not really part of the module. Can you prefix them with underscore?

Reply:
@gtsystem Yes, done! I went back and forth on it when I moved them.